src/olympia/editors/tasks.py

```diff
@@ -158,3 +160,16 @@ def reject_rereview(theme):
     if reupload.footer:
         storage.delete(reupload.footer_path)
     rereview.delete()
+
+
+@task
+@write
+def recalculate_post_review_weight(ids):
+    """Recalculate the post-review weight that should be assigned to
+    auto-approved add-on versions from a list of ids."""
+    addons = Addon.objects.filter(id__in=ids)
+    for addon in addons:
+        for summary in AutoApprovalSummary.objects.filter(
+                version__in=addon.versions.all()):
```

Comment:
This could be improved with something like

```python
summaries = AutoApprovalSummary.objects.filter(
    version__in=addons.versions.all())

for summary in summaries:
    # ...
```